specification/2.0/README.md

```diff
@@ -251,7 +251,7 @@ For property names, glTF uses [camel case](http://en.wikipedia.org/wiki/CamelCas
 
 ## Coordinate System and Units
 
-glTF uses a right-handed coordinate system, that is, the cross product of +X and +Y yields +Z. glTF defines +Y as up. The front of a glTF asset faces +Z.
+glTF uses a right-handed coordinate system; that is, the cross product of right and forward yields up. glTF defines +Y as up, +Z as forward, and -X as right. The front of a glTF asset faces +Z.
```

Comment:
While this statement is technically correct, it's hard to parse because the "right" direction is defined after it's used.

Reply:
Removing the "the cross product ..." would eliminate the confusion without affecting normative definitions.

Reply:
Missed this comment! Done.